Examples/PfRingExample-FilterTraffic/main.cpp

```diff
@@ -326,7 +326,7 @@ int main(int argc, char* argv[])
 			case 'p':
 			{
 				srcPortToMatch = atoi(optarg);
-				if (srcPortToMatch <= 0)
+				if (atoi(optarg) <= 0)
 				{
 					EXIT_WITH_ERROR_AND_PRINT_USAGE("Source port to match isn't a valid TCP/UDP port");
 				}

```

Comment:
Not cool to call atoi() two times IMO maybe something like this ?
```suggestion
				int ret = atoi(optarg);
				if (ret <= 0)
				{
					EXIT_WITH_ERROR_AND_PRINT_USAGE("Source port to match isn't a valid TCP/UDP port");
				}
				srcPortToMatch = ret;
```